numpy/core/src/umath/ufunc_object.c

```diff
@@ -1025,6 +1029,13 @@ get_ufunc_arguments(PyUFuncObject *ufunc,
             }
 
             switch (str[0]) {
+                case 'a':
+                    /* possible axis argument for generalized ufunc */
+                    if (strcmp(str, "axis") == 0 && ufunc->core_enabled) {
+                        *out_axis = value;
```

Comment:
That was captured by the `ufunc->core_enabled`, but I don't know why I didn't just go for the clarity of checking that `out_axis != NULL` (which is also done for `where`).